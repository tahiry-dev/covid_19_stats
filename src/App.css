*,
*::before,
*::after {
  font-family: "Roboto", sans-serif;
  padding: 0;
  margin: 0;
  box-sizing: border-box;
}

body {
  background-color: rgba(253, 83, 148, 0.089);
}

h1 {
  color: #007bff;
  text-align: center;
  padding-top: 30px;
}

.myContainer {
  display: inline-block;
  width: 50%;
  padding-left: 3.5%;
  padding-top: 5%;
  padding-bottom: 5%;
}

.accordion {
  padding-top: 13%;
  width: 80%;
}

h3 {
  color: green;
  padding-top: 5%;
  padding-bottom: 3%;
}

.loading,
.details {
  display: block;
  width: 100%;
  text-align: center;
}

.theStats {
  display: block;
  width: 28%;
  padding-top: 20px;
}

em {
  color: violet;
}

.homeLink {
  display: block;
  margin: 10% auto;
}

.continent_container {
  display: block;
  margin: 0 auto;
}

.continent_form {
  display: block;
  width: 50%;
  margin: 7% auto;
}

.continent_label {
  display: block;
  text-align: center;
}
