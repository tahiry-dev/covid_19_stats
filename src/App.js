import './App.css';

function App() {
  return (
    <div>
      <h1>Hello Covid_stats</h1>
    </div>
  );
}

export default App;
