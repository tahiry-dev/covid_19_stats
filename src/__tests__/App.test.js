
import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';

describe('App Component', () => {
    it('renders pages correctly', () => {
        const tree = renderer.create(<App />);
        expect(tree).toMatchSnapshot();
    });
});

