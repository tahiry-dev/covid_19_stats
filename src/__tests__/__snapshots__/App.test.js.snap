// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders pages correctly 1`] = `
Array [
  <h1>
    CORONA VIRUS DISEASE 19 (COVID-19) STATISTICS
  </h1>,
  <div>
    <div />
  </div>,
]
`;
